AppCheck/Sources/Core/APIService/GACAppCheckAPIService.m

```diff
@@ -40,8 +38,7 @@
 @interface GACAppCheckAPIService ()
 
 @property(nonatomic, readonly) NSURLSession *URLSession;
-@property(nonatomic, readonly) NSString *APIKey;
-@property(nonatomic, readonly) NSString *appID;
+@property(nonatomic, readonly, nullable) NSString *APIKey;
```

Comment:
why is APIKey nullable?

Reply:
The new endpoint that the GSI SDK will use won't require an API key (just the OAuth Client ID). The Firebase App Check SDK will still need to provide an API key though.